mautic/mautic
# Shuffle.js doesn't work

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8312
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR fixes issues with shuffle.js.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Go to the Plugins page. You will see a JS error in the console: 
```js
jQuery(...).shuffle is not a function
```
You have to be on `3.x` branch to reproduce this.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Repeat step 1 from "Steps to reproduce the bug".
3. You will not see the error in the console.
